obs/error.go

```diff
@@ -0,0 +1,22 @@
+package obs
+
+import (
+	"encoding/xml"
+	"fmt"
+)
+
+type Status struct {
+	XMLName xml.Name `json:"status" xml:"status"`
+	Code    string   `json:"code" xml:"code,attr"`
+	Summary string   `json:"summary" xml:"summary"`
+}
+
+type APIError struct {
+	HTTPStatusCode int
+	XMLStatusCode  string
```

Comment:
What do you think about calling this something like `OBSStatusCode` or just `StatusCode`?

Reply:
`OBSStatusCode` sounds good to me.